src/Event/ReadModel/JSONLD/EventLDProjector.php

```diff
@@ -120,14 +122,16 @@ public function __construct(
         SerializerInterface $mediaObjectSerializer,
         IriOfferIdentifierFactoryInterface $iriOfferIdentifierFactory,
         CdbXMLImporter $cdbXMLImporter,
-        JsonDocumentMetaDataEnricherInterface $jsonDocumentMetaDataEnricher
+        JsonDocumentMetaDataEnricherInterface $jsonDocumentMetaDataEnricher,
+        EventFilterInterface $eventFilter
```

Comment:
$eventFilter should be renamed, its purpose is not clear from the current name. $eventsNotTriggeringModifiedUpdate or something similar would be more clear.

I'd also suggest renaming "EventFilterInterface" to "EventSpecification". If something includes "Filter" in its name, I'd rather expect something that reduces a collection/array.